src/extensions/additional/Math/view-and-edit.ts

```diff
@@ -231,7 +231,7 @@ export class MathBlockNodeView extends MathNodeView {
         dom.appendChild(contentDOM);
 
         const hintRendererItem = this.reactRenderer.createItem('math-block-hint', () =>
-            renderMathHint({offset: {left: -3}}, mathHintContainerDOM),
+            renderMathHint({}, mathHintContainerDOM),
```

Comment:
@d3m1d0v check it plz

Reply:
I have checked, no changes are required.